released/packages/coq-equations/coq-equations.1.2+8.8/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Cyprien Mangin <cyprien.mangin@m4x.org>" ]
+dev-repo: "git://github.com/mattam82/Coq-Equations.git"
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://mattam82.github.io/Coq-Equations"
+bug-reports: "https://github.com/mattam82/Coq-Equations/issues"
+license: "LGPL 2.1"
+synopsis: "A function definition package for Coq"
+description: """
+Equations is a function definition plugin for Coq, that allows the
+definition of functions by dependent pattern-matching and well-founded,
+mutual or nested structural recursion and compiles them into core
+terms. It automatically derives the clauses equations, the graph of the
+function and its associated elimination principle.
+"""
+flags: light-uninstall
```

Comment:
Thanks for the conversion, but now the package fails linting due to this line - please remove it (and also for other package).

Reply:
Right, I blindly copied from `coq-bignums.8.9.0` which had it